src/Parser/Monk/Brewmaster/Modules/Features/Checklist.js

```diff
@@ -78,6 +80,24 @@ class Checklist extends CoreChecklist {
         ];
       },
     }),
+    new Rule({
+      name: (
```

Comment:
You should only use the Wrapper component when combining strings and JSX. In all other cases please use strings or template literals.